docs/manual/manual_compiler.html

```diff
@@ -937,6 +938,70 @@ <h3>Compiling Symbolic Circuits<a class="headerlink" href="#compiling-symbolic-c
 <p>Every <code class="xref py py-class docutils literal notranslate"><span class="pre">Backend</span></code> requires <code class="xref py py-class docutils literal notranslate"><span class="pre">NoSymbolsPredicate</span></code>, so it is necessary to instantiate all symbols before running a <code class="xref py py-class docutils literal notranslate"><span class="pre">Circuit</span></code>.</p>
 </div>
 </section>
+<section id="customised-passes">
+<h3>Customised Passes<a class="headerlink" href="#customised-passes" title="Permalink to this heading"></a></h3>
+<p>We have already seen that pytket allows users to combine passes in a desired order using <code class="xref py py-class docutils literal notranslate"><span class="pre">SequencePass</span></code>. An addtional feature is the <code class="xref py py-class docutils literal notranslate"><span class="pre">CustomPass</span></code> allowing users to define their own customised circuit transformation using pytket.
```

Comment:
"allowing users to define their own customised circuit transformation using pytket" -> ", which allows users to define their own customised circuit transformation using pytket"